scripts/icon_studio/controllers/icon_creator/appearances.json

```diff
@@ -11,9 +11,6 @@
   "CarpetFibers": {
     "fields": "colorOverride 0.33 0.34 0.33 textureTransform TextureTransform { scale 1 0.5 }"
   },
-  "ChequeredParquetry": {
-    "fields": "textureTransform TextureTransform { scale 5 2 }"
-  },
```

Comment:
Even if they are deprecated you should leave them here, they should be removed once actually removed.